poker/src/pages/MainPage/MainPage.jsx

```diff
@@ -1,18 +1,42 @@
 import React from 'react';
 import m from './MainPage.module.scss';
 import MainPageTitle from '../../components/MainPageTitle/MainPageTitle';
+import BlueButton from '../../components/BlueButton/BlueButton';
+import InputComponent from '../../components/InputComponent/InputComponent';
 
-const MainPage = () => (
-  <div className={m.MainPage_wrap}>
-    <MainPageTitle />
-    <h3>Start your planning:</h3>
-    <label>Create session:</label>
-    <button>Start new game</button>
-    <h3>OR:</h3>
-    <label>Connect to lobby by URL:</label>
-    <input />
-    <button>Connect</button>
-  </div>
-);
+const MainPage = () => {
+  const handleClick1 = () => {
+    console.log('start new game');
+  };
+  const handleClick2 = () => {
+    console.log('connect');
+  };
+  const handleChange = (e) => {
+    console.log(e.target.value);
+  };
+
+  return (
+    <div className={m.MainPage_wrap}>
+      <MainPageTitle />
+      <h3>Start your planning:</h3>
+      <div className={m.MainPage_item}>
+        <label>Create session:</label>
+        <BlueButton
+          label={'Start new game'}
+          onClick={handleClick1}
+          primary={1}
```

Comment:
magic number. You need to find what is primary and what is 1

Reply:
eslint gives an error, that in props should be no boolean value, and with number-value everything works

Reply:
primary is the color. I can rename it with a different name like 'primaryBG'

Reply:
> eslint gives an error, that in props should be no boolean value, and with number-value everything works

do you try to use just primary in props?

Reply:
it helped